packages/sil-governance/src/components/Header/index.tsx

```diff
@@ -1,19 +1,48 @@
 import { Address } from "@iov/bcp";
 import { Electorate } from "@iov/bns";
 import { Block, CircleImage, Image, Typography } from "medulas-react-components";
-import React from "react";
+import React, { useMemo } from "react";
 
 import iovLogo from "../../assets/iov-logo-title.svg";
 import userIcon from "../../assets/user.svg";
+import { history } from "../../routes";
+import { SHOW_ELECTORATE_ROUTE } from "../../routes/paths";
 
 export const HEADER_HTML_ID = "header";
 
+const navigateToElectorate = (id: number): void => {
+  const target = `${SHOW_ELECTORATE_ROUTE}/${id}`;
+  if (history.location.pathname !== target) {
+    history.push(target);
+  }
+};
+
 interface Props {
   readonly address: Address;
   readonly electorates: readonly Electorate[];
 }
 
 const Header = ({ address, electorates }: Props): JSX.Element => {
+  const ElectorateLinks = useMemo(
+    () => (): JSX.Element => (
+      <React.Fragment>
+        {electorates.map((e, index) => (
+          <Block display="inline" key={e.id}>
```

Comment:
You can archive this with CSS as well (see the very first example in the question of https://stackoverflow.com/questions/27836769/comma-separated-spans-with-pure-css). With CSS, you only need one JSX element per link, allowing you to also drop the container `<Block`